blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -312,6 +386,18 @@ Nav.propTypes = {
       label: 'Horizontal Links hierarchy',
       group: 'Configure content',
     }),
+    desktopNavivationStartHeight: PropTypes.number.tag({
+      label: 'Starting desktop navigation bar height',
+      group: 'Logo',
+      max: 200,
+      min: 56,
+      defaultValue: 56,
```

Comment:
my bad, it should be : default: 56

Reply:
even using `default: 56` it's still not working

Reply:
I think this is correct as `defaultValue` based on these docs:
![image](https://user-images.githubusercontent.com/26662906/103795210-507a8080-500b-11eb-84d4-d0a38c9062a6.png)

With that being said, the `defaultValue` only applies when you add a new instance of a block to the page. So if you had an existing Nav on the page _before_ making this code change, the `defaultValue` won't be picked up and applied to the existing instance of the block. For backward compatibility, we need to make sure that a default value is set even for an existing Nav block so that will need to be set in the code (in addition to keeping it in the custom field `tag()`).

Reply:
Since it could not work as expected, i make another PR to fix the default value for that : https://github.com/WPMedia/fusion-news-theme-blocks/pull/669